### PR TITLE
update sql statement show analyze status

### PR DESCRIPTION
**Pull request by hawkingrei**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/13854
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by Oreoxmt** [comment]
@chrysan @songrijie PTAL

File: sql-statements/sql-statement-show-analyze-status.md [hunk 1]
```diff
@@ -50,16 +52,17 @@ mysql> analyze table t;
...
+| test         | t          | p0             | analyze index idx                                                  |              0 | 2022-05-27 11:29:46 | 2022-05-27 11:29:46 | finished | NULL        | 127.0.0.1:4000 | NULL       | NULL               | NULL  | NULL                 |
+| test         | t          | p1             | analyze columns                                                    |              0 | 2022-05-27 11:29:46 | 2022-05-27 11:29:46 | finished | NULL        | 127.0.0.1:4000 | NULL       | NULL               | NULL  | NULL                 |
+| test         | t          | p0             | analyze columns                                                    |              0 | 2022-05-27 11:29:46 | 2022-05-27 11:29:46 | finished | NULL        | 127.0.0.1:4000 | NULL       | NULL               | NULL  | NULL                 |
++--------------+------------+----------------+--------------------------------------------------------------------+----------------+---------------------+---------------------+----------+-------------+----------------+------------+--------------------+----------+----------------------+
+
```
Comment (Oreoxmt) [inline sql-statements/sql-statement-show-analyze-status.md:65]:
```suggestion
```

**Review by Oreoxmt — APPROVED** [review]
/lgtm

File: information-schema/information-schema-analyze-status.md [hunk 2]
```diff
@@ -19,22 +19,25 @@ DESC analyze_status;
...
+| START_TIME           | datetime            | YES  |      | NULL    |       |
+| END_TIME             | datetime            | YES  |      | NULL    |       |
+| STATE                | varchar(64)         | YES  |      | NULL    |       |
+| FAIL_REASON          | longtext            | YES  |      | NULL    |       |
+| INSTANCE             | varchar(512)        | YES  |      | NULL    |       |
+| PROCESS_ID           | bigint(64) unsigned | YES  |      | NULL    |       |
+| Remaining_seconds    | bigint(64) unsigned | YES  |      | NULL    |       |
+| Progress             | varchar(20)         | YES  |      | NULL    |       |
+| Estimated_total_rows | bigint(64) unsigned | YES  |      | NULL    |       |
++----------------------+---------------------+------+------+---------+-------+
```
Comment (qiancai) [inline information-schema/information-schema-analyze-status.md:39]: 代码里新增的字段名是否也需要和其他字段一样用大写字母呢？

File: information-schema/information-schema-analyze-status.md [hunk 3]
```diff
@@ -44,16 +47,16 @@ SELECT * FROM information_schema.analyze_status;
...
+| test | t | p0 | analyze table all columns with 256 buckets, 500 topn, 1 samplerate | 0 | 2022-05-27 11:30:12 | 2022-05-27 11:30:12 | finished | NULL | 127.0.0.1:4000 | NULL | NULL | NULL | NULL |
+| test | t | p1 | analyze index idx | 0 | 2022-05-27 11:29:46 | 2022-05-27 11:29:46 | finished | NULL | 127.0.0.1:4000 | NULL | NULL | NULL | NULL |
+| test | t | p0 | analyze index idx | 0 | 2022-05-27 11:29:46 | 2022-05-27 11:29:46 | finished | NULL | 127.0.0.1:4000 | NULL | NULL | NULL | NULL |
+| test | t | p1 | analyze columns | 0 | 2022-05-27 11:29:46 | 2022-05-27 11:29:46 | finished | NULL | 127.0.0.1:4000 | NULL | NULL | NULL | NULL |
+| test | t | p0 | analyze columns | 0 | 2022-05-27 11:29:46 | 2022-05-27 11:29:46 | finished | NULL | 127.0.0.1:4000 | NULL | NULL | NULL | NULL |
++--------------+------------+----------------+--------------------------------------------------------------------+----------------+---------------------+---------------------+----------+-------------+----------------+------------+----------------------+----------+-----------------------+-------------------------+--------------------------+
 6 rows in set (0.00 sec)
 ```
```
Comment (qiancai) [inline information-schema/information-schema-analyze-status.md:61]: 是否可以修改下对齐，目前看起来有点乱
image

File: information-schema/information-schema-analyze-status.md (hunk 2 above)
Comment (Oreoxmt) [reply information-schema/information-schema-analyze-status.md:39]: 这三个新增的字段在 playground 上测试发现不是大写字母，请 @hawkingrei 判断一下是否需要修改代码呢

File: information-schema/information-schema-analyze-status.md [hunk 4]
```diff
@@ -44,16 +47,16 @@ SELECT * FROM information_schema.analyze_status;
...
-| test         | t          | p1             | analyze columns                                                    |              0 | 2022-05-27 11:29:46 | 2022-05-27 11:29:46 | finished | NULL        | 127.0.0.1:4000 |       NULL |
-| test         | t          | p0             | analyze columns                                                    |              0 | 2022-05-27 11:29:46 | 2022-05-27 11:29:46 | finished | NULL        | 127.0.0.1:4000 |       NULL |
-+--------------+------------+----------------+--------------------------------------------------------------------+----------------+---------------------+---------------------+----------+-------------+----------------+------------+
++--------------+------------+----------------+--------------------------------------------------------------------+----------------+---------------------+---------------------+----------+-------------+----------------+------------+----------------------+----------+-----------------------+-------------------------+--------------------------+
```
Comment (qiancai) [inline information-schema/information-schema-analyze-status.md:50]: 表头和表的内容对应来看，是否表头下面的行里少了一列
image

File: information-schema/information-schema-analyze-status.md (hunk 2 above)
Comment (hawkingrei) [reply information-schema/information-schema-analyze-status.md:39]: Done

File: information-schema/information-schema-analyze-status.md (hunk 3 above)
Comment (hawkingrei) [reply information-schema/information-schema-analyze-status.md:61]: Done

File: sql-statements/sql-statement-show-analyze-status.md [hunk 5]
```diff
@@ -33,14 +35,14 @@ mysql> analyze table t;
 Query OK, 0 rows affected (0.20 sec)
```
[the remark this answers is not in the document]
Comment (hawkingrei) [reply sql-statements/sql-statement-show-analyze-status.md:36]: Done

File: information-schema/information-schema-analyze-status.md [hunk 6]
```diff
@@ -44,16 +47,16 @@ SELECT * FROM information_schema.analyze_status;
...
++--------------+------------+----------------+--------------------------------------------------------------------+----------------+---------------------+---------------------+----------+-------------+----------------+------------+----------------------+----------+-----------------------+
+| TABLE_SCHEMA | TABLE_NAME | PARTITION_NAME | JOB_INFO                                                           | PROCESSED_ROWS | START_TIME          | END_TIME            | STATE    | FAIL_REASON | INSTANCE       | PROCESS_ID | REMAINING_SECONDS    | PROGRESS | ESTIMATED_TOTAL_ROWS  |
++--------------+------------+----------------+--------------------------------------------------------------------+----------------+---------------------+---------------------+----------+-------------+----------------+------------+----------------------+----------+-----------------------+
+| test         | t          | p1             | analyze table all columns with 256 buckets, 500 topn, 1 samplerate |              0 | 2022-05-27 11:30:12 | 2022-05-27 11:30:12 | finished |        NULL | 127.0.0.1:4000 | NULL       | NULL                 | NULL     |                  NULL |
```
Comment (chrysan) [inline information-schema/information-schema-analyze-status.md:53]: 需要给一个带 REMAINING_SECONDS    | PROGRESS | ESTIMATED_TOTAL_ROWS   具体信息的例子

File: information-schema/information-schema-analyze-status.md [hunk 7]
```diff
@@ -70,3 +73,6 @@ SELECT * FROM information_schema.analyze_status;
 * `FAIL_REASON`：任务失败的原因。如果执行成功则为 `NULL`。
 * `INSTANCE`：执行任务的 TiDB 实例。
 * `PROCESS_ID`：执行任务的 process ID。
+* `REMAINING_SECONDS`：表示任务预计还需要多少秒才能完成。
+* `PROGRESS`：表示任务的进度。
+* `ESTIMATED_TOTAL_ROWS`：表示任务需要分析的总行数。
```
Comment (hfxsd) [inline information-schema/information-schema-analyze-status.md:78]:
```suggestion
* `REMAINING_SECONDS`：执行任务预计还需要多少秒才能完成。
* `PROGRESS`：执行任务的进度。
* `ESTIMATED_TOTAL_ROWS`：执行任务需要分析的总行数。
```

**Review by hfxsd — APPROVED** [review]

File: information-schema/information-schema-analyze-status.md (hunk 6 above)
Comment (hawkingrei) [reply information-schema/information-schema-analyze-status.md:53]: Done

**Review by chrysan — APPROVED** [review]

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**